### ISSUE TITLE
Bug: Outdated group invite link preview

### ISSUE THREAD

**Issue by saulmc** [ephemeraHQ/converse-app#965]
### Describe the bug

A group invite link was created. Later, the group name, photo, and descriptions were changed. After those changes, the link preview image and metadata still reflects the older group metadata.

### Expected behavior

If group metadata changes, so should its preview link metadata.

### Steps to reproduce the bug

Create a group named X
Create an invite link to group X
Change the group name to Y
Share the invite link in Slack or Converse
See the group still appears as X in the invite link preview

**Comment by alexrisch** [comment]
@saulmc the backend get's no information to the group information, whenever the group meta data changes 

If we really wanted to we can check each time a user loads into the app and update, but I feel like these will be some limitations

**Comment by saulmc** [comment]
Could the app inform the server when the user saves a metadata update, similar to how it tells the server when the group is first created?

**Comment by alexrisch** [comment]
We can, right now it's pretty heavily tied to the creator of the group invite, it can get pretty messy with our current architecture though

I would say this is fairly high effort to do correctly, and could be fairly low return
The user should be able to delete the invite and recreate one if they so wanted to

**Comment by saulmc** [comment]
Agreed, and invite deletion is already a higher priority for security reasons.
